homeassistant/components/wled/config_flow.py

```diff
@@ -57,7 +57,7 @@ async def async_step_zeroconf(
         )
 
         # Prepare configuration flow
-        return await self._handle_config_flow(discovery_info, True)
+        return await self._handle_config_flow(cast(dict, discovery_info), True)
```

Comment:
I think that it would be better to explicitely keep only the required elements but I think it is out of scope for this PR.